src/Fixer/PhpUnit/PhpUnitTestAnnotationFixer.php

```diff
@@ -305,7 +305,7 @@ private function addTestPrefix($functionName)
             return 'test_'.$functionName;
         }
 
-        return'test'.ucfirst($functionName);
+        return 'test'.ucfirst($functionName);
```

Comment:
Do we not have a fixer for this?

Reply:
almost https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4435 :)